### PR TITLE
Add rule namespace:meter_pvc_bytes_total:sum

### PR DESCRIPTION
**Pull request by yunkunrao**

### CODE REVIEW

File: roles/ks-monitor/files/prometheus/prometheus-rules-v1.16+.yaml [hunk 1]
```diff
@@ -725,6 +725,9 @@ spec:
     - expr: |
         sum(label_replace(label_join(kube_pod_container_resource_requests * on (pod, namespace)group_left(owner_kind,owner_name)label_replace(label_join(label_replace(label_replace(kube_pod_owner{job="kube-state-metrics"},"owner_kind","Deployment","owner_kind","ReplicaSet"),"owner_kind","Pod","owner_kind","<none>"),"tmp",":","owner_name","pod"),"owner_name","$1","tmp","<none>:(.*)"),"workload",":","owner_kind","owner_name"),"workload","$1","workload","(Deployment:.+)-(.+)")) by (namespace, workload, resource)* on(namespace) group_left(workspace) kube_namespace_labels{job="kube-state-metrics"}
       record: namespace:kube_workload_resource_request:sum
+    - expr: |
+        max by (namespace, node, persistentvolumeclaim) (kubelet_volume_stats_capacity_bytes) * on (namespace, persistentvolumeclaim) group_left (storageclass) kube_persistentvolumeclaim_info{} * on(namespace) group_left(workspace) kube_namespace_labels{job="kube-state-metrics"}
```
Comment (benjaminhuo) [inline roles/ks-monitor/files/prometheus/prometheus-rules-v1.16+.yaml:729]: kube_persistentvolumeclaim_info{} 
=>
kube_persistentvolumeclaim_info{job="kube-state-metrics"}

File: roles/ks-monitor/files/prometheus/prometheus-rules-v1.16+.yaml [hunk 2]
```diff
@@ -725,6 +725,9 @@ spec:
...
+      record: namespace:meter_pvc_bytes_total:sum
```
Comment (benjaminhuo) [inline roles/ks-monitor/files/prometheus/prometheus-rules-v1.16+.yaml:730]: namespace:meter_pvc_bytes_total:sum
=>
namespace:pvc_bytes_total:

**Review by benjaminhuo — CHANGES_REQUESTED** [review]
Pls see my comments above

File: roles/ks-monitor/files/prometheus/prometheus-rules-v1.16+.yaml (hunk 1 above)
Comment (yunkunrao) [reply roles/ks-monitor/files/prometheus/prometheus-rules-v1.16+.yaml:729]: got it

File: roles/ks-monitor/files/prometheus/prometheus-rules-v1.16+.yaml (hunk 2 above)
Comment (yunkunrao) [reply roles/ks-monitor/files/prometheus/prometheus-rules-v1.16+.yaml:730]: got it

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**